### PR TITLE
Feature add remove

### PR DESCRIPTION
**Pull request by NatanimA**
> Inserted Todo list project structure

> Inserted Add and Remove functions

> Inserted Line-through checker

> Inserted Edit function

> Removal by Selection only by pressing the trash button currently as the requirement

> Followed Html and CSS best practices

> Inserted webpack and loaders to the project

> Fetching of data dynamically by Javascript objects

`Please do this to run it on your pc`

`npm start` =>this will prompt you to install webpack then `y` and it will install

`npm run build` => this will generate codes to the dist folder

`npm start` and this will start the server and you can see the the project I did 😉

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,59 +1,34 @@
...
-        ${add.completed ? 'complete' : ''}>
-        <p> ${add.description}</p>
-        <i class="fa-solid fa-ellipsis-vertical"></i>`;
-
-      toDoList.appendChild(task);
+import TasksObject from './modules/tasksObject.js';
+import WindowLoader from './modules/windowsLoader.js';
+import LocalDataStorage from './modules/localStorage.js';
+import CreateTasks from './modules/createTasks.js';
+
```
Comment (SajjadAhmad14) [inline src/index.js:8]: Please add a function for editing task descriptions. Currently, the user is not able to edit the task and on refresh, it's the same old task item.

File: src/modules/createTasks.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        content.classList.toggle('line-through');
+        trash.classList.toggle('trash-active');
+        dots.classList.toggle('dots-remove');
+
+        const localData = JSON.parse(localStorage.getItem('todo'));
+        const filter = [];
+        const taskChecked = document.querySelectorAll('.todo-list-container');
+        for (let i = 0; i < localData.length; i += 1) {
+          if (taskChecked[i].classList.contains('checked-holder')) {
+            localData[i].completed = true;
```
Comment (SajjadAhmad14) [inline src/modules/createTasks.js:30]: User when checking an item the `completed` value doe not become `true` in local storage. Kindly make sure that the value gets `true` on checked and `false` on unchecked.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Natanim, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by GzimAsani — COMMENTED** [review]
Hi @NatanimA , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has already been merged. I understand that you did that by mistake, but please open new PR and leave the link for this old PR where the next reviewer can see the required changes asked by the previous reviewer

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by EricMbouwe — COMMENTED** [review]
Hi @NatanimA, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Requested changes ✔
- Linters ✔

## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**